documentation/build.sbt

```diff
@@ -45,7 +46,7 @@ val apiDoc =
       ),
       unidocProjectFilter in(ScalaUnidoc, unidoc) := inProjects(
         `algebra-jvm`, `algebra-circe-jvm`, `algebra-playjson-jvm`,
-        `akka-http-client`, `akka-http-server`,
+        `akka-http-client`, `akka-http-server`, `akka-http-server-circe`, 
```

Comment:
We should also add `akka-http-server-playjson` here.

Reply:
done